docs/content/dagster-cloud/deployment/serverless.mdx

```diff
@@ -37,14 +37,47 @@ Enterprise customers may request a quota increase by [contacting Sales](mailto:s
 
 ## Getting started with Serverless
 
-When you create a new Dagster Cloud organization, you'll need to activate a deployment type: Hybrid or Serverless. Once activated, you'll be prompted to fork the starter GitHub repo and install the GitHub app. Pushing to the main branch will deploy to your prod Serverless deployment. Pull requests will spin up ephemeral [branch deployments](/dagster-cloud/developing-testing/branch-deployments) using the Serverless agent.
+### With GitHub
 
-If you’re using another git hosting provider, CI service, or simply want to do local development without using a CI service, you can use Serverless via the [dagster-cloud CLI](/dagster-cloud/developing-testing/dagster-cloud-cli). Run the following to learn more:
+If you are a GitHub user, our GitHub integration is the fastest way to get started. It uses a GitHub app and GitHub Actions to set up a repo containing skeleton code and configuration consistent with Dagster Cloud's best practices with a single click.
 
+When you create a new Dagster Cloud organization, you'll be prompted to choose Serverless or Hybrid. Once activated, our GitHub integration will scaffold a new git repo for you with serverless and branch deployments preconfigured. Pushing to the `main` branch will deploy to your `prod` Serverless deployment. Pull requests will spin up ephemeral [branch deployments](/dagster-cloud/developing-testing/branch-deployments) using the Serverless agent.
+
+### Without GitHub (GitLab, BitBucket, or local development)
+
+If you don't want to use our GitHub integration, we offer a powerful CLI that you can use in another CI environment or on your local laptop.
+
+First, let's [create a new project](https://docs.dagster.io/getting-started/create-new-project#create-a-new-project) with the Dagster open-source CLI.
```

Comment:
```suggestion
First, [create a new project](https://docs.dagster.io/getting-started/create-new-project#create-a-new-project) with the Dagster open-source CLI.
```

Suggesting we remove `let's` here. The rest of the instructions in this section use an implied subject/understood you, so making this one consistent feels less awkward to me.